src/core/environment.js

```diff
@@ -369,6 +369,35 @@ p5.prototype.noCursor = function() {
   this._curElement.elt.style.cursor = 'none';
 };
 
+/**
+ * If the was created in WebGL mode, then `weglVersion` will indicate which
+ * version of WebGL it is using. It will try to create a WebGL2 canvas unless
+ * you have set `p5.disableWebGL2 = true`, and will fall back to WebGL1 if
+ * WebGL2 is not available.
+ *
+ * `webglVersion` will always be either `WEBGL2` or `WEBGL`.
+ * @property {String} webglVersion
+ * @readOnly
+ * @example
+ * <div><code>
+ * let myFont;
+ * function preload() {
+ *   myFont = loadFont('assets/inconsolata.otf');
+ * }
+ * function setup() {
```

Comment:
Is there any other way to illustrate the difference between webGL1 vs 2 aside from swapping out a string based on the version?

Reply:
Maybe we could swap out two different shaders based on the webgl version, where one uses GLSL 2 and the other 3? It might be a bit verbose as an example though.

Reply:
I was hoping there would be a simple visual example. I like the idea of showing the different shaders, maybe it would help people to see the differences side by side like that.

Reply:
Most of the features seem to be behind-the-scenes changes that you'd see in the code more than in the output. Looking at this https://webgl2fundamentals.org/webgl/lessons/webgl2-whats-new.html maybe the biggest visual change would be support for repeated textures of arbitrary sizes? Where in v1 it would fall back to clamp but in v2 would repeat?